docs/sources/explore/explore-inspector.md

```diff
@@ -0,0 +1,82 @@
++++
+title = "Inspector in Explore"
+weight = 400
++++
+
+# Inspector in Explore
+
+The inspector helps you understand and troubleshoot your queries. You can inspect the raw data, export that data to a comma-separated values (CSV) file, export log results in TXT format, and view query requests.
+
+## Inspector UI
+
+The inspector consists of four tabs:
+
+- **Data tab -** Shows the raw data returned by the query.
+- **Stats tab -** Shows how long your query takes and how much it returns.
+- **JSON tab -** Allows you to view and copy the data JSON and data frame structure JSON.
+- **Query tab -** Shows you the requests to the server sent when Grafana queries the data source.
```

Comment:
List them as they show in the UI - Stats, Query inspector, JSON, and Data

Reply:
Fixed in bd89ecf. I have also added Error tab as https://github.com/grafana/grafana/pull/33412 adds it.